test/parallel_api/ranges/copy_if_ranges_sycl.pass.cpp

```diff
@@ -0,0 +1,73 @@
+// -*- C++ -*-
```

Comment:
What's exactly name do you suggest?

Reply:
I couldn't imagine the best name. It could be `copy_if_remove_copy_if_ranges_sycl`, but I don't really like it. `remove_copy_if_ranges_sycl` hides `copy_if` algo. Unfortunately, I couldn't imagine anything better...

Reply:
`copy_if_remove_copy_if_ranges_sycl`  - double "copy_if" funs and confuses ..
I would suggest keeping "copy_if...." , especially `copy_if`, `remove_copy`, `remove_copy_if `are one algorithm essentially.